app/models/AppComponents.scala

```diff
@@ -0,0 +1,21 @@
+package models
+import play.api.ApplicationLoader.Context
+import play.api.BuiltInComponentsFromContext
+import play.api.db.Database
+import play.api.db.DBComponents
+import play.api.db.HikariCPComponents
+import play.api.db.evolutions.EvolutionsComponents
+import play.api.routing.Router
+import play.filters.HttpFiltersComponents
+
+class AppComponents(cntx: Context)
```

Comment:
I see now, this should be an abstract class and you can remove the override below

Reply:
Ah alright, got it!